plugins/woocommerce/src/Blocks/BlockTypes/ProductButton.php

```diff
@@ -132,13 +137,14 @@ protected function render( $attributes, $content, $block ) {
 			* @param number $default_quantity The default quantity.
 			* @param number $product_id The product id.
 			*/
+			// Question: Is this info in the Store API cart?
```

Comment:
Question: Is this info in the Store API cart?

Reply:
It's not the same. It looks like this is a [legacy filter](https://github.com/woocommerce/woocommerce/blob/579ecb010843352573c1810168b16e1da03018d3/docs/cart-and-checkout-blocks/hooks/migrated-hooks.md).